### ISSUE TITLE
Integrate with discord permission nodes

### ISSUE THREAD

**Issue by lekro** [lekro/elosensei#3]
* Require a particular rank/role/permission node to add matches/administrate
* Restrict bot's replies to certain channels
* This should be configurable
* Nicely ask for permissions to change user's roles if that's configured
* Think about configuration for roles (by name, by id?)

**Comment by chr1s-b** [comment]
This could be done with a function decorator around the commands. E.g
```Python
@commands.command
@role('Elo Manager')
def match(self,ctx):
    ...
    return
```

**Comment by lekro** [comment]
Implemented in 2c2ccfb7542216961f60629e8ffae6af76727de2. The permissions configuration is also in the configuration file.
